controllers/dashboard/routes.controller.js

```diff
@@ -129,6 +129,7 @@ function RoutesController(options) {
       }
     ], function(err, result) {
       if (err)
+        // TODO: this wont work, self is not defined
         return self.destroy(err);
```

Comment:
This comes from another part of code.. It is a mistake, nothing to destroy here!